lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -428,6 +428,35 @@ struct ExprVisitor {
     return builder.create<moore::ExtractOp>(loc, type, value, lowBit);
   }
 
+  Value visit(const slang::ast::MemberAccessExpression &expr) {
+    llvm::SmallVector<const slang::ast::ValueSymbol *> symbolVec;
+    const slang::ast::Expression *exprIt = &(expr.value());
+    std::string concatName = {};
+    std::string memberName(expr.member.name);
+    concatName = memberName + concatName;
```

Comment:
`memberName` is just used once, so I think we can declare `concatName` with `expr.member.name`.

Reply:
Good advice!